apps/users/tests.py

```diff
@@ -37,13 +38,13 @@ class UserLoginTest(StaticLiveServerTestCase):
     def setUpClass(cls):
         super().setUpClass()
         chrome_options = Options()
-        # runs headless chrome for wsl environments
-        if (
-            "linux" in platform.uname()[0].lower()
-            and "microsoft" in platform.uname()[2].lower()
-        ):
+        if settings.SYSTEM == "wsl":
```

Comment:
Can we change these away from using "SYSTEM" and create a "headless" and an "sandbox" env variable to separately configure this testing feature?

Reply:
New CHROME_HEADLESS and CHROME_SANDBOX settings now hard-coded in ci.py